src/components/EntityStatus/EntityStatus.scss

```diff
@@ -1,7 +1,9 @@
 @use '../../styles/mixins.scss';
 
 .entity-status {
-    --button-width: 28px;
+    --button-width: 24px;
+    --entity-status-height: 20px;
```

Comment:
Without fixed height this element for some reason has extra 4px empty height. It was not visible because of 5px margin on link element. It can be fixed by display `flex` / `inline-flex`, but such display break buttons layout.

`EntityStatus` without fixed height and with `display:block`:
<img width="271" height="45" alt="Screenshot 2025-09-04 at 15 36 02" src="https://github.com/user-attachments/assets/c630dc72-54c4-4e62-b17f-69e4af3911a8" />